data/avro-schema/src/main/resources/avro/net/corda/data/membership/rpc/certificates/request/CertificateRpcRequest.avsc

```diff
@@ -0,0 +1,20 @@
+{
+  "type": "record",
+  "name": "CertificateRpcRequest",
+  "namespace": "net.corda.data.membership.rpc.certificates.request",
```

Comment:
Maybe there is a better package for this since it isn't strictly membership related. i.e. cluster level certificates are a level below membership.

Reply:
Discussed in stand up.  
I would suggest either leave it as is in `membership` package, despite the cluster level certs not being directly connected to membership, or create a `certificates` package. e.g. `net.corda.data.certificates`.
Any preference @yift-r3 ?